modules/apps/commerce/commerce-account-item-selector-web/src/main/resources/META-INF/resources/account_user_item_selector.jsp

```diff
@@ -0,0 +1,127 @@
+<%--
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+CommerceAccountUserItemSelectorViewDisplayContext commerceAccountUserItemSelectorViewDisplayContext = (CommerceAccountUserItemSelectorViewDisplayContext)request.getAttribute(WebKeys.PORTLET_DISPLAY_CONTEXT);
+
+SearchContainer<User> commerceAccountUserSearchContainer = commerceAccountUserItemSelectorViewDisplayContext.getSearchContainer();
+String itemSelectedEventName = commerceAccountUserItemSelectorViewDisplayContext.getItemSelectedEventName();
+PortletURL portletURL = commerceAccountUserItemSelectorViewDisplayContext.getPortletURL();
+%>
+
+<liferay-frontend:management-bar
```

Comment:
@andrea-ale-sbarra this is an old pattern that is deprecated please update with latest